extension.js

```diff
@@ -17,29 +17,32 @@ function activate(context) {
 	const TykSchemas = [
 		{
 			"fileMatch": [
-				"tyk.*.conf"
+				"tyk.conf",
+				"tyk-*.conf"
 			],
 			"url": "https://raw.githubusercontent.com/TykTechnologies/tyk-schemas/v0.1/JSON/draft-07/schema_tyk.oss.conf",
 			"addedBy": "tyk"
 		},
 		{
 			"fileMatch": [
-				"apikey.*.json"
+				"apikey-*.json"
 			],
 			"url": "https://raw.githubusercontent.com/TykTechnologies/tyk-schemas/v0.1/JSON/draft-07/schema_apikey.json",
 			"addedBy": "tyk"
 		},
 		{
 			"fileMatch": [
-				"apidef.*.json",
-				"TykDefinition-*.json"
+				"apidef.json",
+				"apidef-*.json",
+				"TykAPIDef-*.json"
```

Comment:
Is it API or Api? I see both in oas and the classic definition. Are we inconsistent?

Reply:
@buraksekili

Reply:
@letzya it is `API` because of our because of internal discussion. I can update it if you wish